ports/stb/vcpkg.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "stb",
+  "version-string": "2021-06-20",
```

```suggestion
  "version-date": "2021-06-26",
```
This should be [version-date](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-date). We should use the date that the commit was accessed by you.